default_data.json

```diff
@@ -38274,11 +38274,6 @@
             "organization": "istio",
             "uri": "https://github.com/istio/proxy.git"
         },
-        {
-            "module": "istio.github.io",
-            "organization": "istio",
-            "uri": "https://github.com/istio/istio.github.io.git"
-        },
```

Comment:
I'm good with the other changes, but why you removed this one?

Reply:
this repo has bean renamed to https://github.com/istio/istio.io, thanks.